### ISSUE TITLE
Upon clicking IconButton label appears and it will only disappear when we click somewhere else on the screen

### ISSUE THREAD

**Issue by krishnasreekoganti** [carbon-design-system/ibm-security#900]
### Description

IconButton label appearing is not consistent over different browsers. I am currently using Chrome and I see when I click on the IconButton the label appears and stays back on the screen until I click somewhere else on the screen.

<img width="469" alt="Screenshot 2021-01-28 at 10 31 26 PM" src="https://user-images.githubusercontent.com/19775110/106240517-66561c80-622a-11eb-990e-61d6d9ad5286.png">

#### Expected behavior

When I hover on IconButton I can see the label and it goes off when I un-hover. Similarly it should disappear after clicking on the IconButton. This is the scenario with safari & Firefox.

#### Actual behavior

In Chrome when I click on the IconButton the label appears and only disappears when I click somewhere else on the screen.

### Steps for reproducing

https://ibm-security.carbondesignsystem.com/?path=/story/components-iconbutton--default

1. Open the above link in Crome
2. Click on the IconButton label appears and stay on the screen 
3. Label only disappears only when I click somewhere else on the screen 

### Affected browsers

Chrome on Mac OSX 10 ---> issue is replicated
Firefox on Mac OSX 10 ---> issue is not replicated
Safari on Mac OSX 10 ---> issue is not replicated

**Comment by SimonFinney** [comment]
Thanks for raising! I believe this may be related to how each of the browsers interprets whether a button element should receive focus when it has been selected.

Does the following information reflect the behaviour you've observed above? https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#Clicking_and_focus

**Comment by krishnasreekoganti** [comment]
Yes that's the same behaviour I faced, Is there any way I can avoid that in Chrome?

**Comment by SimonFinney** [comment]
I believe it might be possible by intercepting the event in JavaScript, but I don't recommend altering the browser's default behaviour
